WooCommerce/Classes/ViewRelated/Dashboard/DashboardViewController.swift

```diff
@@ -271,6 +271,24 @@ private extension DashboardViewController {
         })
     }
 
+    private func handleSyncError(error: Error) {
```

Comment:
This looks great. The UI handles an error that is generated by the only layer it is aware of: the business logic